vulture/core.py

```diff
@@ -29,6 +29,7 @@
 import argparse
 import ast
 from fnmatch import fnmatchcase
+from lxml import etree
```

Comment:
Yes, it is an external dependency. I see two major advantages of why we should be using `lxml`:
* It provides a neat, easy and robust API for `xpath` lookup.
* The `xml` module which ships with `stdlib` is not secure against malicious data.

Reply:
The stdlib XML module also supports XPath (https://docs.python.org/2/library/xml.etree.elementtree.html#elementtree-xpath) and we don't have to worry about malicious XML input, I think, since users should be able to trust their own input. I'd really like to avoid an external dependency, so could you please try to use only the stdlib?

Reply:
Sure thing. Let's give it a try.